lib/pure/random.nim

```diff
@@ -632,6 +662,17 @@ when isMainModule:
       elif oc > 150:
         doAssert false, "too many occurrences of " & $i
 
+    when false:
+      var rs: RunningStat
+      for j in 1..5:
+        for i in 1 .. 1_000:
+          rs.push(gauss())
+        echo("mean: ", rs.mean,
+          " stdDev: ", rs.standardDeviation(),
+          " min: ", rs.min,
+          " max: ", rs.max)
```

Comment:
Can we improve this test?

For example, we could `assert` things about the properties of the generated distribution. But I don't know what Nim's policy is on probabilistic tests.

Reply:
python does the same.

Reply:
use sane bounds and let it fail once in a billion years ;-)

Reply:
@ee7 @timotheecour Do you want to come up with some tests yourself? Because I can't come up with something that will not fail.

Reply:
I'd prefer to let someone else handle it. But I'll echo @timotheecour when he wrote 
> - turn when false into enabled code, otherwise it doesn't even check that it keeps compiling/working

and that it'd be nice to have a goodness-of-fit test (see e.g. https://en.wikipedia.org/wiki/Normality_test) especially if we create a module in `fusion` and start adding other distributions.

Reply:
please give me some codes, its hard for me to implement the tests by myself

Reply:
I'm sorry, but I don't have the time right now.

But given that the consensus seems to be that we'll put this in a new module in `fusion`, perhaps there isn't a big rush to add the best tests.

Furthermore, if we end up adding more distributions then it'd probably be good to discuss and tweak the tests anyway. It's not the most shallow topic.

Reply:
thanks for the understanding